GNUmakefile

```diff
@@ -73,6 +73,15 @@ tests-lint-fix: tools
 	@find ./kubernetes -name "*_test.go" -exec sed -i ':a;N;$$!ba;s/fmt.Sprintf(`\n/fmt.Sprintf(`/g' '{}' \; # remove newlines for terrafmt
 	@terrafmt fmt -f ./kubernetes --pattern '*_test.go'
 
+test-update: fmtcheck vet
+	rm -rf $(EXT_PROV_DIR)/.terraform $(EXT_PROV_DIR)/.terraform.lock.hcl || true
+	mkdir $(EXT_PROV_DIR)/.terraform
+	mkdir -p /tmp/.terraform.d/localhost/test/kubernetes/9.9.9/$(OS_ARCH) || true
+	go clean -cache
+	go build -o /tmp/.terraform.d/localhost/test/kubernetes/9.9.9/$(OS_ARCH)/terraform-provider-kubernetes_9.9.9_$(OS_ARCH)
+	cd $(EXT_PROV_DIR) && TF_CLI_CONFIG_FILE=$(EXT_PROV_DIR)/.terraformrc TF_PLUGIN_CACHE_DIR=$(EXT_PROV_DIR)/.terraform terraform init -upgrade
+	TF_CLI_CONFIG_FILE=$(EXT_PROV_DIR)/.terraformrc TF_PLUGIN_CACHE_DIR=$(EXT_PROV_DIR)/.terraform TF_ACC=1 go test $(TEST) -v -run 'regression' $(TESTARGS)
+
```

Comment:
This change makes the `testacc` command faster. If you need to run update-related tests (the ones with `_regression` in the name, now you can run `make test-update`.

It also works with non-update related tests. Sometimes I like to do this if I've made schema changes that I want to be sure are in the test run:

```
TESTARGS="-run 'TestAccKubernetesJob'" make test-update
```